.gitignore

```diff
@@ -1,23 +1,61 @@
-# Compiled class file
-*.class
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and WebStorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
-# Log file
-*.log
+# User-specific stuff
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/dictionaries
+.idea/**/shelf
 
-# BlueJ files
-*.ctxt
+# Sensitive or high-churn files
+.idea/**/dataSources/
+.idea/**/dataSources.ids
+.idea/**/dataSources.local.xml
+.idea/**/sqlDataSources.xml
+.idea/**/dynamic.xml
+.idea/**/uiDesigner.xml
+.idea/**/dbnavigator.xml
 
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
+# Gradle
+.idea/**/gradle.xml
+.idea/**/libraries
 
-# Package Files #
-*.jar
-*.war
-*.nar
-*.ear
-*.zip
-*.tar.gz
-*.rar
+# CMake
+cmake-build-debug/
+cmake-build-release/
 
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
+# Mongo Explorer plugin
+.idea/**/mongoSettings.xml
+
+# File-based project format
+*.iws
+
+# IntelliJ
+out/
+
+# mpeltonen/sbt-idea plugin
+.idea_modules/
+
+# JIRA plugin
+atlassian-ide-plugin.xml
+
+# Cursive Clojure plugin
+.idea/replstate.xml
+
+# Crashlytics plugin (for Android Studio and IntelliJ)
+com_crashlytics_export_strings.xml
+crashlytics.properties
+crashlytics-build.properties
+fabric.properties
+
+# Custom
+.idea/httpRequests
+.idea/
```

Comment:
compiler.xml, workspace.xml (basically the whole .idea folder) and Cucumber.json (target folder) are still being tracked as modified files, since they are present in Git. To remove them please use the next command - once they are not in Git, they will be ignored correctly:
<img width="658" alt="image" src="https://user-images.githubusercontent.com/90613721/220401780-577cb4e3-bc5a-4fc0-9497-8063912ae692.png">